client/my-sites/checkout/checkout/composite-checkout.js

```diff
@@ -654,56 +665,138 @@ function getCheckoutEventHandler( dispatch ) {
 				return dispatch(
 					recordTracksEvent( 'calypso_checkout_composite_step_changed', { step: action.payload } )
 				);
-			case 'STRIPE_TRANSACTION_BEGIN':
+			case 'STRIPE_TRANSACTION_BEGIN': {
+				dispatch(
+					recordTracksEvent( 'calypso_checkout_form_submit', {
+						credits: null,
+						payment_method: 'WPCOM_Billing_Stripe_Payment_Method',
+					} )
+				);
 				return dispatch(
 					recordTracksEvent( 'calypso_checkout_composite_stripe_submit_clicked', {} )
 				);
-			case 'STRIPE_TRANSACTION_ERROR':
+			}
+			case 'STRIPE_TRANSACTION_ERROR': {
+				dispatch(
+					recordTracksEvent( 'calypso_checkout_payment_error', {
+						error_code: null,
```

Comment:
Do you think we should add something for `error_code`?

Reply:
I do, although the error_code appears to be undefined in the live data. The main reason I left it alone is that it's not clear where the error code is on the error object here.